src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -67,6 +68,12 @@ object ScalafixPlugin extends AutoPlugin {
         "Optional list of custom rules to install from Maven Central. " +
           "This setting is read from the global scope so it only needs to be defined once in the build."
       )
+    val scalafixScalaBinaryVersion: SettingKey[String] =
+      settingKey[String](
+        "The Scala binary version used for scalafix execution. Defaults to 2.12. " +
+          s"Rules must be compiled against that binary version. Advanced semantic rules such as " +
+          s"ExplicitResultTypes require this to match the Scala version used for compilation."
```

Comment:
The last sentence may be a bit misleading since it indicates this version must match the scala version exactly (MAJOR.MINOR.PATCH), not just the binary version (MAJOR.MINOR).

Reply:
Indeed, I am rephrasing to
```
        "The Scala binary version used for scalafix execution. Defaults to 2.12. " +
          s"Rules must be compiled against that binary version, or for advanced rules such as " +
          s"ExplicitResultTypes which have a full cross-version, against the corresponding full" +
          s"version that scalafix is built against."
```